src/rest-server/src/models/v2/user.js

```diff
@@ -22,6 +22,24 @@ const groupModel = require('@pai/models/v2/group');
 
 const crudType = 'k8sSecret';
 const crudUser = crudUtil.getStorageObject(crudType);
+const keygen = require('ssh-keygen');
+const deasync = require('deasync');
+
+function getSSHKey() {
+  let ret;
+  const location = 'id_rsa';
+  keygen({ location: location }, function (err, result) {
+    if (err) {
+      console.log(err);
+    }
+    ret = result;
+  });
+  // eslint-disable-next-line no-unmodified-loop-condition
```

Comment:
Remove this line. Please **DO NOT** disable eslint unless you have a good reason